samples/k8s-existing-cluster-only-sample/manifest.yaml

```diff
@@ -0,0 +1,13 @@
+applicationName: com.contoso.azurevote.existing
```

Comment:
Is this file required in this sample folder?

Reply:
may be  yes, to depict the changes are only in those files. I see other samples also has the same so keeping that for consistency.